data/json/recipes/electronics/tools.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "result": "makeshift_oscilloscope",
+    "type" : "recipe",
+    "category": "CSC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_TOOLS",
+    "skill_used": "electronics",
+    "difficulty": 4,
+    "skills_required": [ "fabrication", 2 ],
+    "time": 240000,
+    "book_learn": [ [ "advanced_electronics", 4 ], [ "mag_electronics", 6 ] , [ "manual_electronics", 5 ] ],
+    "qualities": [
+      { "id": "SCREW_FINE", "level": 1 },
+      { "id": "MULTIMETER", "level": 2 }
+    ],
+    "tools": [ [ [ "soldering_iron", 20 ], [ "toolset", 20 ] ] ],
+    "components": [
+      [ [ "television", 1 ] ],
```

Comment:
Btw. the description of the television says "A large cathode ray tube television, full of delicious electronics." and disassembling it gives *no* LCD screen or similar. This should be kept in sync.

The TV is either CRT or LCD based (I don't care), but its usage in any other recipe, its disassemble recipe and its description should match.